tests/pytests/unit/utils/win_dacl/test_get_sid.py

```diff
@@ -0,0 +1,45 @@
+"""
+tests.pytests.unit.utils.win_dacl.test_get_sid
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Test the get_sid function in the win_dacl utility module
+"""
+# Python libs
+import pytest
+
+# Salt libs
+import salt.utils.win_dacl
+
+# Third-party libs
+try:
+    import pywintypes
+    import win32security
+
+    HAS_WIN32 = True
+except ImportError:
+    HAS_WIN32 = False
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+]
+
+
+@pytest.mark.skipif(not HAS_WIN32, reason="Requires Win32 libraries")
+def test_get_sid_string():
+    """
+    Validate getting a pysid object from a name
```

Comment:
same here, it looks like there are a lot of similarities you could make use of `@pytest.mark.parametrize` and slim down to one test.

Reply:
Done